test/__snapshots__/preset-attributify.test.ts.snap

```diff
@@ -102,22 +135,44 @@ exports[`attributify > fixture1 1`] = `
 
 exports[`attributify > fixture2 1`] = `
 "/* layer: default */
+[pointer-events-none=\\"\\"]{pointer-events:none;}
+[relative=\\"\\"]{position:relative;}
+[absolute=\\"\\"]{position:absolute;}
+[left-4=\\"\\"]{left:1rem;}
+[top~=\\"\\\\31 \\\\/3\\"]{top:33.33333333333333%;}
+[m-0=\\"\\"]{margin:0rem;}
 [ma=\\"\\"]{margin:auto;}
 [mb--3=\\"\\"]{margin-bottom:-0.75rem;}
+[mb-a=\\"\\"]{margin-bottom:auto;}
 [mt-1=\\"\\"]{margin-top:0.25rem;}
+[mt-a=\\"\\"]{margin-top:auto;}
+[h-14=\\"\\"]{height:3.5rem;}
 [h-80=\\"\\"]{height:20rem;}
-[flex=\\"\\"]{display:flex;}
+.flex,[flex=\\"\\"]{display:flex;}
+[inline-flex=\\"\\"]{display:inline-flex;}
+[flex-col=\\"\\"]{flex-direction:column;}
+[peer=\\"\\"]:focus~[peer-focus~=\\"-translate-y-4\\"],[peer=\\"\\"]:not(:placeholder-shown)~[peer-not-placeholder-shown~=\\"-translate-y-4\\"],[peer=\\"\\"]:focus~[peer-focus~=\\"scale-75\\"],[peer=\\"\\"]:not(:placeholder-shown)~[peer-not-placeholder-shown~=\\"scale-75\\"]{--un-rotate:0;--un-scale-x:1;--un-scale-y:1;--un-scale-z:1;--un-skew-x:0;--un-skew-y:0;--un-translate-x:0;--un-translate-y:0;--un-translate-z:0;transform:rotate(var(--un-rotate)) scaleX(var(--un-scale-x)) scaleY(var(--un-scale-y)) scaleZ(var(--un-scale-z)) skewX(var(--un-skew-x)) skewY(var(--un-skew-y)) translateX(var(--un-translate-x)) translateY(var(--un-translate-y)) translateZ(var(--un-translate-z));}
+[peer=\\"\\"]:focus~[peer-focus~=\\"-translate-y-4\\"],[peer=\\"\\"]:not(:placeholder-shown)~[peer-not-placeholder-shown~=\\"-translate-y-4\\"]{--un-translate-y:-1rem;}
+[peer=\\"\\"]:focus~[peer-focus~=\\"scale-75\\"],[peer=\\"\\"]:not(:placeholder-shown)~[peer-not-placeholder-shown~=\\"scale-75\\"]{--un-scale-x:0.75;--un-scale-y:0.75;}
+[peer=\\"\\"]:focus~[peer-focus~=\\"origin-top-left\\"],[peer=\\"\\"]:not(:placeholder-shown)~[peer-not-placeholder-shown~=\\"origin-top-left\\"]{transform-origin:top left;}
 [select-none=\\"\\"]{user-select:none;}
+[px-4=\\"\\"]{padding-left:1rem;padding-right:1rem;}
+[pt-4=\\"\\"]{padding-top:1rem;}
 [text-center=\\"\\"]{text-align:center;}
-[text-4xl=\\"\\"]{font-size:2.25rem;line-height:2.5rem;}
+[peer=\\"\\"]:checked~[peer-checked~=\\"text-4xl\\"],[text-4xl=\\"\\"]{font-size:2.25rem;line-height:2.5rem;}
 [text-5xl=\\"\\"]{font-size:3rem;line-height:1;}
 [font-100=\\"\\"]{font-weight:100;}
 [font-200=\\"\\"]{font-weight:200;}
+[leading-1rem=\\"\\"]{line-height:1rem;}
 [tracking-wider=\\"\\"]{letter-spacing:0.05em;}
-.group:hover [group-hover~=\\"text-teal-400\\"]{--un-text-opacity:1;color:rgba(45,212,191,var(--un-text-opacity));}
-.group:hover [group-hover~=\\"op-50\\"]{opacity:0.5;}
+[group=\\"\\"]:hover [group-hover~=\\"text-teal-400\\"]{--un-text-opacity:1;color:rgba(45,212,191,var(--un-text-opacity));}
```

Comment:
Even when attributify is enabled, I think matching the class version of `.group` and `.peer` is till needed as people might mix them together